packages/suite-base/src/panels/RawMessages/types.ts

```diff
@@ -11,6 +11,8 @@ export enum NodeState {
   Expanded = "e",
 }
 
+export const PATH_NAME_AGGREGATOR = "~";
```

Comment:
Please, move this constant to a proper file `RawMessages/constants.ts`.
And also move "Constants" constant at line 28.

- [ ] move PATH_NAME_AGGREGATOR to constants.ts
- [ ] move "...Constants" to constants.ts

Reply:
and please, when moving "Constants", do not reuse this same obj structure. Create one const for each one.